src/App/ComponentsDocumentation/components/Tables/index.js

```diff
@@ -36,13 +36,13 @@ const Overview = () => (
             </div>
             <p>By default, most column data is left aligned. There could be two exceptions which should be right aligned to help users identify number size and interactions.</p>
             <div className="row">
-                <div className="col-6">
+                <div className="col-lg-6 col-sm-12">
```

Comment:
why `col-sm-12`  and not `col-12` so it starts from viewport 0px wide?
(it applies to all other changes of this file too)

Reply:
Fair point 🤓 Just copy/pasted from Button Doc.... busted.......